CTOR-945: Update EN version an add a french version for sc-service-now-em-events (#4031)
centreon/centreon-documentation: versioned_docs/version-24.10/integrations/event-management/sc-service-now-em-events.md

2. Click on **central-broker-master** (or the appropriate broker configuration if it is a poller or a remote server that will send events).
3. On the **Output** tab, select **Generic - Stream connector** from the list and then click **Add**. A new output appears in the list.
4. Fill in the fields as follows:

| Field           | Value                                                         |
| --------------- | ------------------------------------------------------------- |
| Name            | Servicenow events                                             |
| Path            | /usr/share/centreon-broker/lua/servicenow-em-events-apiv2.lua |
| Filter category | Neb                                                           |

5. To enable Centreon to connect to your ServiceNow Event Manager equipment, fill in the following mandatory parameters. 
The fields for the first entry are already present. Click on the **+Add a new entry** link located below the **Filter category** table to add another one.

| Type   | Name          | Value explanation                    | Value exemple |
| ------ | ------------- |--------------------------------------| ------------- |

> | string | instance      | The name of the service now instance | MyCompany     |

Reviewer: ```suggestion
| string | instance      | The name of the ServiceNow instance | MyCompany     |
```

Author: ServiceNow is the product name in this case
![image](https://github.com/user-attachments/assets/d8f2f142-8a46-4d4c-bdad-72b4766ce2d4)

| string | client_id     | The Oauth client_id                  |               |
| string | client_secret | The Oauth client_secret              |               |
| string | username      | The Oauth user                       |               |

> | string | password      | The Oauth pasword                    |               |

Reviewer: ```suggestion
| string | password      | The Oauth password                    |               |
```

6. Fill in any optional parameters you want (using the **+Add a new entry** link):

| Type   | Name      | Value explanation                          | default value                                               |
| ------ | --------- |--------------------------------------------| ----------------------------------------------------------- |
| string | logfile   | The file in which logs are written         | /var/log/centreon-broker/servicenow-em-stream-connector.log |
| number | log_level | Logging level from 1 (errors) to 3 (debug) | 1                                                           |

7. Use the stream connector's optional parameters to [filter or adapt the data you want Centreon to send to ServiceNow Event Manager](#filtering-or-adapting-the-data-you-want-to-send-to-servicenow-em).

8. [Deploy the configuration](../../monitoring/monitoring-servers/deploying-a-configuration.md).

9. Restart **centengine** on all pollers:

   ```shell
   systemctl restart centengine
   ```

>    ServiceNow Event Manager should now receive data from Centreon. To test if it is working, see [Curl commands: testing the stream connector](#curl-commands-testing-the-stream-connector).

Reviewer: ```suggestion
ServiceNow Event Manager should now receive data from Centreon. To test if it is working, see [Curl commands: testing the stream connector](#curl-commands-testing-the-stream-connector).
```
